polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -2669,6 +2669,7 @@ class MLIRAction : public clang::ASTFrontendAction {
       : Module(Module), ModuleId(ModuleId) {
     EmitIfFound.insert(Fn);
   }
+  bool hasIRSupport() const final { return true; }
```

Comment:
We did not have a need to override this member function. Can you explain the reason we need to override it now ?

Reply:
Not needed, just forgot to remove after doing some tests